tools/dataMigration/migrateAreas.ts

```diff
@@ -3,17 +3,26 @@ import { BaseProtocol } from '../../server/db'
 type Props = {
   client: BaseProtocol
   schema: string
+  index: number
 }
 
 export const migrateAreas = async (props: Props): Promise<void> => {
-  const { client, schema } = props
+  const { client, schema, index } = props
 
   await client.query(`
-        insert into ${schema}.country (country_iso)
-        select country_iso
-        from country
-        order by country_iso;
-    `)
+      insert into ${schema}.country (country_iso, props)
+      select
+          c.country_iso,
+          c.config || jsonb_build_object(
+              'status',  ${index === 0 ? 'a.status' : `'editing'`},
+              'desk_study',  ${index === 0 ? 'a.desk_study' : false},
+              'forestCharacteristics', jsonb_build_object('useOriginalDataPoint', false)
+              ) as props
```

Comment:
this should be migrated from old tables

Reply:
What is the old value or where should I find this?

Reply:
this is in table public.dynamic_country_configuration.
move this table to _legacy schema and use _legacy.dynamic_country_configuration in migrate-fra.
what do you think?

<img width="600" alt="Screenshot 2022-04-06 at 16 27 21" src="https://user-images.githubusercontent.com/1138156/161998125-e3a11304-f3af-4040-9b6c-22424bc85ad8.png">